sky/skylet/constants.py

```diff
@@ -50,6 +50,9 @@
 # Docker default options
 DEFAULT_DOCKER_CONTAINER_NAME = 'sky_container'
 DEFAULT_DOCKER_PORT = 10022
+DOCKER_USERNAME_ENV_KEY = 'DOCKER_USERNAME'
+DOCKER_PASSWORD_ENV_KEY = 'DOCKER_PASSWORD'
+DOCKER_REPO_URI_ENV_KEY = 'DOCKER_REPO_URI'
```

Comment:
Should we set the env vars with the prefix `SKYPILOT_` to avoid being conflicted with the user's existing env vars.